distribution/client/src/lib/status.ts

```diff
@@ -38,8 +48,8 @@ class Status {
    * Create a status record in the backend for this particular instance
    */
   async create() {
-    let api = this.app.service('status');
-    let record = {
+    const api = this.app.service('status');
```

Comment:
AIUI, these `let` => `const` are unrelated to the typescript migration, and were done incidentally while looking at this code. Or does typescript enforce this? (I'll check/discover that later anyway :))

Reply:
Just cleaning up while I was here :smile: